testpilot/frontend/static-src/styles/modules/_main-header.scss

```diff
@@ -1,6 +1,12 @@
 #main-header {
   @include flex-container(row, space-between, center);
 
+  // overrides the page view inserting the header
+  // on the landing page
+  [data-hook=landing-page] &:nth-child(2) {
```

Comment:
This would probably be better done in the view JS, just don't add the elements to the DOM rather than hide them after the fact